src/main/java/org/embulk/input/zendesk/services/ZendeskCustomObjectService.java

```diff
@@ -0,0 +1,109 @@
+package org.embulk.input.zendesk.services;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.http.HttpStatus;
+import org.embulk.config.ConfigException;
+import org.embulk.config.TaskReport;
+import org.embulk.input.zendesk.RecordImporter;
+import org.embulk.input.zendesk.ZendeskInputPlugin;
+import org.embulk.input.zendesk.clients.ZendeskRestClient;
+import org.embulk.input.zendesk.models.Target;
+import org.embulk.input.zendesk.models.ZendeskException;
+import org.embulk.input.zendesk.stream.paginator.CustomObjectSpliterator;
+import org.embulk.input.zendesk.utils.ZendeskConstants;
+import org.embulk.input.zendesk.utils.ZendeskUtils;
+import org.embulk.spi.Exec;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class ZendeskCustomObjectService implements ZendeskService
+{
+    protected ZendeskInputPlugin.PluginTask task;
+
+    private ZendeskRestClient zendeskRestClient;
+
+    public ZendeskCustomObjectService(final ZendeskInputPlugin.PluginTask task)
+    {
+        this.task = task;
+    }
+
+    @VisibleForTesting
+    protected ZendeskRestClient getZendeskRestClient()
```

Comment:
fixed

Reply:
hmm, why don't you move `protected` down to `public` instead of changing it to public? Also please scan through source code for other classes too, e.g. https://github.com/treasure-data/embulk-input-zendesk/blob/821f7c9a5a933cc5350842d4ee85f6bce9c15f94/src/main/java/org/embulk/input/zendesk/services/ZendeskNormalServices.java#L43-L52 or https://github.com/treasure-data/embulk-input-zendesk/blob/2843e6d771ad38ddb147003335eb1244db1e0f78/src/main/java/org/embulk/input/zendesk/services/ZendeskUserEventService.java#L41-L48.

I'd recommend move it down to public methods instead of changing modifier from `protected` to `public` though.